public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+# Our First Ever Truffle Retreat
+
+![Truffle](/blog/images/first-ever-truffle-retreat/medium-truffle-retreat.jpg)
+
+The Truffle team recently got together late September to define our roles, our mission statement, and our product roadmap - and had some fun in the process.  
```

Comment:
Wasn't it late August?

Suggest, "This past summer, the Truffle team met together in person for the first time to define our roles, our mission statement, and our product roadmap..."

Reply:
✔️